framework/test/jest/unit/specs/application/node/synchronizer/block_synchronization_mechanism/block_synchronization_mechanism.spec.js

```diff
@@ -1156,22 +1133,20 @@ describe('block_synchronization_mechanism', () => {
 
 	describe('isValidFor', () => {
 		it('should return true if the difference in block slots between the current block slot and the finalized block slot of the system is bigger than ACTIVE_DELEGATES*3', async () => {
-			when(storageMock.entities.Block.getOne)
+			when(storageMock.entities.Block.get)
```

Comment:
this also should be changed to blocks mock?

Reply:
I think we can keep as storage Mock, because in the end the blocks is calling storage.

Reply:
I think to separate the concern, and not not affecting the changes from `lisk-blocks`, it's better not to use the storage.
If we keep it, when we change the internal implementation, it will be affected

Reply:
Since this will introduce huge change, we will create a separate issue to address this